### DISCUSSION TITLE
Ability to change Difficulty range slider with arrow keys

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#20415] [Ideas]
Add the ability to change these points:
![image](https://user-images.githubusercontent.com/31819497/191932484-c8e357a3-efb0-4f3b-94c0-f2381a804a9e.png)
with the arrow keys like you can use arrow keys for fine adjustments of sliders in the settings.

**Comment by peppy** [comment]
While I agree with making this a thing, currently priority is given to the song select key bindings (select next / select previous group).

It's not a super-simple one to fix.
